README.md

```diff
@@ -56,13 +56,12 @@ metadata:
   name: read-configmaps
 rules:
 - verbs:
-  - watch
-  - get
-  - list
+  - '*'
   apiGroups:
   - ""
   resources:
   - configmaps
+  - pods
```

Comment:
I'm wondering why we need these extra permissions? It was not needed for first Rust implementation.

Reply:
```  go

	err = leader.Become(context.Background(), controllerName)
	if err != nil {
		return err
	}
```

this code uses it. I'll comment it.